src/MeasureHelpers.ts

```diff
@@ -403,3 +403,14 @@ export function extractMeasurementPeriod(measureBundle: R4.IBundle): Calculation
 function __guard__(value: any, transform: any) {
   return typeof value !== 'undefined' && value !== null ? transform(value) : undefined;
 }
+/**
+ * Figure out if a string is a  valid URL  or if it's just a string
+ * @public
+ * @param {string} libraryName - The name of the library depending on the origin of the measure this may a url
+ * @return {boolean} If the statement is a function or not.
+ */
+export function isValidLibraryURL(libraryName: string) {
```

Comment:
We may want to add some positive/negative tests for this, if possible; something like calling `isValidLibraryURL` on `https://example.com/Library-url` and `Library/example` or the like, and ensuring the results are what we expect.